### PR TITLE
feat: added command service

### PR DESCRIPTION
**Pull request by tstaetter**
command service to be used by the UI. Each action on the UI should invoke one of the command available in this command service. The API will then create a corresponding command row and issue the appropriate BE command

### CODE REVIEW

File: blockjoy/api/ui_v1/command_service.proto [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+message CommandResponse {
+  ResponseMeta meta = 1;
+}
+
+/* Service providing commands invoked by the UI */
+service CommandService {
+  /* "Create node" command */
+  rpc CreateNode(CommandRequest) returns (CommandResponse) {}
```
Comment (loony-bean) [inline blockjoy/api/ui_v1/command_service.proto:20]: I recall @chrisabruce wanted to have a generic command, especially if all params and results are the same like here

Comment (tstaetter) [reply blockjoy/api/ui_v1/command_service.proto:20]: CommandRequest is generic, isn't it?

Comment (loony-bean) [reply blockjoy/api/ui_v1/command_service.proto:20]: I mean something like
```
rpc Command(CommandRequest) returns (CommandResponse) {}
```
and no other variants

Comment (tstaetter) [reply blockjoy/api/ui_v1/command_service.proto:20]: It will end up calling fns like Command::create_node etc., so I feel this approach is more expressive

Comment (loony-bean) [reply blockjoy/api/ui_v1/command_service.proto:20]: Yeah but adding support for new commands will require software upgrades

**Review by loony-bean — APPROVED** [review]

**Pull request merged**